action.yml

```diff
@@ -38,15 +38,23 @@ inputs:
 
   source:
     description: |-
-      (Required, unless providing `metadata` or `image`) Path to source to
-      deploy. If specified, this will deploy the Cloud Run service from the code
-      specified at the given source directory.
+      (Required, unless providing `metadata`, `image`, or `job`) Path to source
+      to deploy. If specified, this will deploy the Cloud Run service from the
+      code specified at the given source directory.
+
+      This requires the Artifact Registry API to be enabled. Furthermore, the
+      deploying service account must have the `Cloud Build Service Account`
+      role. The initial deployment will create an Artifact Registry repository
+      which requires the `Artifact Registry Admin` role.
+    required: false
 
-      This requires the [Artifact Registry API][artifact-api] to be enabled.
-      Furthermore, the deploying service account must have the `Cloud Build
-      Service Account` role. The initial deployment will create an [Artifact
-      Registry repository][repo] which requires the `Artifact Registry Admin`
-      role.
+  job:
+    description: |-
+      (Required, unless providing `metadata` or `service`) ID of the job or
+      fully-qualified identifier of the job. If `job` and `service` are
```

Comment:
This text says the service will be updated but the code errors if both are specified?